File: src/FMS.Site/src/components/endSeason/presentation/prorel.js
import React, { PropTypes } from "react";

import Row from "../../table/presentation/row";
import Data from "../../table/presentation/data";

const ProRel = ({ prorel }) => {

    return (
        <Row>
            <Data>{prorel.division}</Data>
            <Data>{prorel.status}</Data>
            <Data>{prorel.team}</Data>
        </Row>
    );
};

ProRel.propTypes = {
    prorel: PropTypes.object.isRequired
};

export default ProRel;

File: src/FMS.Site/Models/ProRel.cs
using FMS.Site.Data;

namespace FMS.Site.Models
{
    public class ProRel
    {
        public int Id { get; set; }
        public int TeamId { get; set; }
        public int DivisionId { get; set; }
        public string Status { get; set; }
        public int SeasonId { get; set; }

        public string Team => TeamData.GetTeamById(TeamId).Name;
        public string Division => DivisionData.GetDivisionById(DivisionId).Name;
    }
}
